### DISCUSSION TITLE
Nuove funzioni nell'aggiornamento?

### DISCUSSION THREAD

**Discussion by scuolagioco** [iisgiua/giuaschool#252] [Idee e proposte]
Salve, quali funzioni saranno implementate nel nuovo aggiornamento?

**Comment by trinko** [comment]
La gestione dello scrutinio finale: devo fare alcune modifiche per la conversione dei crediti per le quinte e, se riesco, comincio ad introdurre qualche modifica per renderlo un più generico e configurabile.

**Comment by ghost** [comment]
@scuolagioco Buongiorno! Avrei bisogno di parlarti, in merito a questa richiesta. Posso parlarti in privato? Mi scrivi alla mia mail: padovesi.roberto@gmail.com? Grazie!
